scalac-scoverage-plugin/src/test/scala/scoverage/CoberturaXmlWriterTest.scala

```diff
@@ -314,7 +317,16 @@ class CoberturaXmlWriterTest
     val writer = new CoberturaXmlWriter(sourceRoot, dir)
     writer.write(coverage)
 
-    val xml = XML.loadFile(fileIn(dir))
+    val customXML: XMLLoader[Elem] = XML.withSAXParser {
+      val factory = SAXParserFactory.newInstance()
+      factory.setFeature(
+        "http://apache.org/xml/features/nonvalidating/load-external-dtd",
+        false
+      )
+      factory.newSAXParser()
+    }
+
+    val xml = customXML.loadFile(fileIn(dir))
```

Comment:
This needed to change in order to account for https://github.com/scala/scala-xml/pull/177. I'm not super worried about this seeing that we are only using this parser to test here.